katheryne/baerritos.py

```diff
@@ -0,0 +1,35 @@
+def constant(f):
+    def fset(self, value):
+        raise TypeError
+    def fget(self):
+        return f()
+    return property(fget, fset)
+
+class BaerritoConstants(object):
+    @constant
+    def BAERRITOS_GUILD_ID():
```

Comment:
why not just `BAERRITOS_GUILD_ID = 689238357670232083` in class scope? seems simpler imo

Reply:
actually, the same in module scope would be plenty. rename the module `baerritos_constants` and you can replace all instances of `BC.` with the module

Reply:
fair enough, let me make the change